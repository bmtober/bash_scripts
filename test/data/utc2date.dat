0
1
1459908160
-283978800
2147483647
2147483648
