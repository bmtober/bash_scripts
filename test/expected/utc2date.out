1969-12-31 19:00:00
1969-12-31 19:00:01
2016-04-05 22:02:40
1961-01-01 00:00:00
2038-01-18 22:14:07
